src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml.cs

```diff
@@ -121,7 +121,8 @@ private void OnNodeAutoCompleteSearchControlVisibilityChanged(object sender, Dep
             Dispatcher.BeginInvoke(new Action(() =>
             {
                 SearchTextBox.Focus();
-                ViewModel.InitializeDefaultAutoCompleteCandidates();
+                //ViewModel.InitializeDefaultAutoCompleteCandidates();
+                ViewModel.PopulateAutoCompleteCandidates();
```

Comment:
We probably want to consider returning the number of filtered results here, if the number is 0 which means we failed to find any match, then we still call `InitializeDefaultAutoCompleteCandidates()`.

Reply:
The reason I commented this out was I noticed issues with this. On a mouse up, the node suggestion list was reverting to this default list each time.